R/estimate_KM.R

```diff
@@ -184,6 +185,15 @@ estimate_KM <- function(
     }
   }
 
+  # add strata labels ----------------------------------------------------------
```

Comment:
Should not be a new section, but rather a comment.

Reply:
if that is your preference, i'll remove